plugins/trivia.lua

```diff
@@ -194,13 +223,15 @@ return {
           .."!trivia to obtain a new question. \n"
           .."!trivia [answer] to answer the question. \n"
           .."!triviaquestion to show the current question. \n"
-          .."!triviascore to get a scoretable of all players.",
+          .."!triviascore to get a scoretable of all players. \n"
+          .."!triviaskip to skip a question (requires sudo)",
```

Comment:
Right - I just threw this together in a couple minutes because my friends were stuck on a question with not text, and I just did whatever was there before. I'll change that in a second!